ui/core.js

```diff
@@ -13,6 +13,8 @@
 //>>description: The core of jQuery UI, required for all interactions and widgets.
 //>>docs: http://api.jqueryui.com/category/ui-core/
 //>>demos: http://jqueryui.com/
+//>>css.structure: ../themes/base/core.css
```

Comment:
Comments got lost, so repeating them here. I said:

> `core.js` and `core.css` are quite different. There's nothing in this JS file that actually depends on any CSS. I'd like to get input from @arschmitz on this, but as far as I can tell, core.css should be referenced as css.structure by all widgets that need it, but not here.

Rafael asked:

> Does any widget depend on core.css, but not on core.js?

I think the answer is "no", but that still seems like its just abusing an accidental relationship. There is no direct relationship between `core.js` and `core.css`, its only through inheritance.

Hoping for input from @arschmitz here as well (again).